source/geod24/LocalRest.d

```diff
@@ -115,11 +125,6 @@ private struct TimeCommand
     bool drop = false;
 }
 
-/// Ask the node to shut down
-private struct ShutdownCommand
```

Comment:
Why did you move it ? Just template it.

Reply:
I thought if I templatized it you would complain about there being too many templates. :D

Reply:
Well it's a template anyway 🤷